src/main/rx/cc2500_frsky_d.c

```diff
@@ -113,10 +113,16 @@ static void frSkyDTelemetryWriteByte(const char data)
 static void buildTelemetryFrame(uint8_t *packet)
 {
     uint8_t a1Value;
-    if (rxFrSkySpiConfig()->useExternalAdc) {
-        a1Value = (adcGetChannel(ADC_EXTERNAL1) & 0xff0) >> 4;
-    } else {
+    switch (rxFrSkySpiConfig()->a1Source) {
+      case FRSKY_SPI_A1_SOURCE_VBAT:
```

Comment:
`switch` and `case` are at the same indentation level in Betaflight (see https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#curly-braces).